5.1.js

```diff
@@ -0,0 +1,4 @@
+var arrays = [[1, 2, 3], [4, 5], [6]];
+console.log(arrays.reduce(function(unified, current) {

```

Comment:
staraj sie tworzyc funkcje a nie console.logi. jezeli bedziesz chciala cos wypisac to potem tylko sconsole.logujesz wynik